packages/css-data/src/property-parsers/text-shadow-property-extractor.ts

```diff
@@ -0,0 +1,55 @@
+import type {
+  KeywordValue,
+  RgbValue,
+  TupleValue,
+  TupleValueItem,
+  UnitValue,
+} from "@webstudio-is/css-engine";
+
+/*
+  extractBoxShadowProperties accepts a TupleValue
+  and just tokenizes the valuess of box-shadow items.
+  https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow#syntax
+
+  To validate the text-shadow string value, should be used
+  in combination with parseShadow function.
+
+  Eg:
+  extractBoxShadowProperties(parseShadow('boxShadow', "5em 5em 5em 5em #ffff"))
+*/
+
+export type ExtractedTextShadowProperties = {
+  color: RgbValue | KeywordValue | null;
```

Comment:
is there any way to not use null? we are generally trying to use undefined all the time and null only where some external api forces us to use null

Reply:
Will switch to `undefined` then 👍